db/db_test.go

```
@@ -55,6 +55,21 @@ func (suite *DBTestSuite) TestFind() {
 	suite.Empty(modules)
 }
 
+func (suite *DBTestSuite) TestDelete() {
```

golangci-lint (1.64.8)

[warning] empty-lines: extra empty line at the end of a block

(revive)